cli/internal/specs/v0/spec_reader.go

```diff
@@ -384,3 +385,17 @@ func stripYamlComments(b []byte) ([]byte, error) {
 	}
 	return b, nil
 }
+
+func shouldEscapeFileContent(content []byte) bool {
+	if !bytes.ContainsAny(content, "\n\r") {
+		return false
+	}
+	if json.Valid(content) {
+		return true
+	}
+	block, _ := pem.Decode(content)
+
+	// if block is nil then content is a valid PEM block and should return true
```

Comment:
pem.Decode:
```
// Decode will find the next PEM formatted block (certificate, private key
// etc) in the input. It returns that block and the remainder of the input. If
// no PEM data is found, p is nil and the whole of the input is returned in
// rest. Blocks must start at the beginning of a line and end at the end of a line.
func Decode(data []byte) (p *Block, rest []byte) {
```

if block is nil, content isn't a valid pem block?

There is also an example in `pem_test.go` in the stdlib that does:
```go
	isEmpty := func(block *Block) bool {
		return block != nil && block.Type == "EMPTY" && len(block.Headers) == 0 && len(block.Bytes) == 0
	}
```

Reply:
I mean, the code is correct, but the comments are opposite.
```suggestion
	// if block is not nil then content is a valid PEM block and should return true
```